chainer/training/extensions/log_report.py

```diff
@@ -56,17 +56,26 @@ class LogReport(extension.Extension):
             formatting. For example, users can use '{iteration}' to separate
             the log files for different iterations. If the log name is None, it
             does not output the log to any file.
+            It is recommended to suffix the filename with ``'.jsonl'`` when
+            ``file_format='jsonl'``.
             For historical reasons ``log_name`` is also accepted as an alias
             of this argument.
+        file_format (str): Format of the log file. If ``'jsonl'`` is specified,
+            a log file is in JSON Lines format, which is JSON objects delimited
+            by newlines (``LF``). It is suitable for stream processing and
```

Comment:
"LF" is not a code fragment.
```suggestion
            by newlines (LF). It is suitable for stream processing and
```